proto/provenance/expiration/v1/expiration.proto

```diff
@@ -0,0 +1,43 @@
+// Since: provenance 1.12
+syntax = "proto3";
+package provenance.expiration.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/expiration/types";
+option java_package        = "io.provenance.expiration.v1";
+option java_multiple_files = true;
+
+// Params defines the set of params for the expiration module.
+message Params {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.stringer)         = true;
+  option (gogoproto.goproto_stringer) = false;
+
+  // the deposit amount for holding module assets on chain
+  cosmos.base.v1beta1.Coin deposit = 1 [(gogoproto.nullable) = false];
+  // duration is the period upon which an expiration will stay on chain
+  // when the period elapses the module asset is up for extension or deletion
+  //
+  // the accepted values for a duration are n{h,d,w,y}
+  // where 1h = 60m, 1d = 24h, 1w = 7d (or 168h), 1y = 365d (or 8760h)
+  string duration = 2;
+}
+
+// Expiration holds a typed key/value structure for data associated with an expiring module asset
+message Expiration {
+  // the module asset identifier
+  string module_asset_id = 1;
+  // The bech32 address the expiration is bound to
+  string owner = 2;
+  // The time the module asset expires
+  google.protobuf.Timestamp time = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+  // The deposit amount held while module asset is in use
+  repeated cosmos.base.v1beta1.Coin deposit = 4
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  // Message relating to the expiring module asset
```

Comment:
I'd like this to have more info. "relating to" doesn't really answer the question, "What is this field for?"